docs/documentation/siddhi-4.0.md

```diff
@@ -2292,6 +2292,10 @@ extension type.
 
 * Follow the procedure for the required archetype, based on your project:
 
+!!! Note
+    When using generated archetypes please make sure you un-comment @Extension annotation and complete the
+    annotation with proper values. This annotation will be used to identify and document extension and hence your
+    extension will not work without @Extension annotation.
```

Comment:
IMO /s/This annotation will be used to identify and document extension and hence your extension will not work without @Extension annotation./This annotation will be used to identify and document **the** extension, hence your extension will not work without @Extension annotation.